py4DSTEM/process/diskdetection/multicorr_row_kernel.cu

```diff
@@ -0,0 +1,60 @@
+#include <cupy/complex.cuh>
+#define PI 3.14159265359
+extern "C" __global__
+void multicorr_row_kernel(
+	complex<float> *ar,
+	const float *xyShifts,
+	const long long N_pts,
+	const long long image_size_x,
+	const long long image_size_y,
+	const long long upsample_factor) {
```

Comment:
Do these need to be `long long`?

Reply:
Yes, the `int` type in Python maps to `long long` in CUDA